addons/mail/static/src/components/emoji_grid_search_no_content_view/emoji_grid_search_no_content_view.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.EmojiGridSearchNoContentView" owl="1">
+        <div class="o_EmojiGridSearchNoContentView d-flex flex-column w-100 h-100 justify-content-center align-items-center">
+            <div class="o_EmojiGridSearchNoContentView_image">😢</div>
```

Comment:
would probably have been better to have an actual image

Reply:
There is no image like this. Here the idea is copy discord, which shows a stylized icon. This is the quick and easy way to have this visual. Also this is a emoji picker, so using an emoji for this visual makes sense

Reply:
ok as a provisional, "quick and easy" solution, but it would be better to have an actual image at some point. Currently, we have no control over the image that is displayed, it can be different from one device to another

Reply:
Yes, although I'd say the problem is kind of coupled with visual inconsistency of emojis. In the sense that here we want to display a sad crying smiley. Emoji are visuals for that. We made the decision to use OS/browser specific visual, so here it's a similar problem.